src/styles/blocks/catalog.scss

```diff
@@ -1,25 +1,149 @@
-.catalog {
+.card-conteiner {
   display: flex;
+  margin: 0 auto;
+  padding: 50px $main-pading-left-right;
   flex-wrap: wrap;
+  width: 1024px;
+}
 
-  max-width: 944px;
-  margin: 0 auto;
-  padding: 50px 0;
+.card {
+  margin: 0;
+  width: $card-width;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: $text-white-color;
+  margin-right: $card-margin-right;
+  margin-bottom: 46px;
+  height: $card-height;
+  transition: transform 300ms;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
 
-  &__card {
+  &__info {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-main-color;
+    transition: 300ms;
+  }
+
+  .info {
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-secondary-color;
+  }
+
+  .code {
```

Comment:
no need to create BEM mix here
it's just BEM element, so you can set `margin` in `&__code`